clusters/build-clusters/cluster-init.yaml

```diff
@@ -0,0 +1 @@
+managed:
```

Comment:
Go ahead and add

```yaml
managed:
- build01
- build02
```

?

Reply:
Yeah, when we have @hongkailiu's PR to allow this to generate the `build01` and `build02` jobs in we should add those. I can do it preemptively.